runtime/process-posix.c

```diff
@@ -558,6 +571,11 @@ stz_int launch_process(stz_byte* file, stz_byte** argvs, stz_int input,
 
   // Child: setup pipes, exec
   else {
+
```

Comment:
I recently read that the child process inherits the parent's signal mask and signal handlers. Do we need to unblock the signal mask in the child process?

Additionally, do we need to uninstall the signal handler in the child process before the exec call?

Reply:
Never mind: I answered my own question. We don't need to do anything special about the signal mask or handler: exec will reset everything itself.

> execve() does not return on success, and the text, data, bss, and stack of the calling process are overwritten by that of the program loaded. The program invoked inherits the calling process's PID, and any open file descriptors that are not set to close on exec. Signals pending on the calling process are cleared. Any signals set to be caught by the calling process are reset to their default behaviour. The SIGCHLD signal (when set to SIG_IGN) may or may not be reset to SIG_DFL.